topnav/assets/img/ia-hamburger-fff.svg

```diff
@@ -0,0 +1,3 @@
+<svg height="40px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
```

Comment:
Question: can we make this a component so we can just pass in their color fills?

Reply:
Are we looking to create a general component for all svgs, or make every svg a component of its own? The latter is doable but the former may raise the problems that we encountered in #179

Reply:
No not yet. Ideally, there should only be 1 hamburger svg, 1 images svg, etc. is there a way to resolve this by either component-izing or css?

Reply:
I've componentized the used svgs as suggested